Code/Framework/AzToolsFramework/AzToolsFramework/Application/ToolsApplication.cpp

```diff
@@ -376,12 +376,6 @@ namespace AzToolsFramework
         EditorAssetMimeDataContainer::Reflect(context);
         ComponentAssetMimeDataContainer::Reflect(context);
 
-        AssetBrowser::AssetBrowserEntry::Reflect(context);
```

Comment:
This has all been removed, as these items no longer use objectstream or the reflect context in any way.  It was being used to (improperly) serialize and deserialize these into the drag and drop event.

Improperly, becuase it would not actually deserialize properly and would leak.